metastore/validator.go

```diff
@@ -142,6 +142,10 @@ func (v tableSchemaValidatorImpl) validateIndividualSchema(table *common.Table,
 		if colIdDedup[colId] {
 			return common.ErrDuplicatedColumn
 		}
+		colDataType :=  memCom.DataTypeFromString(table.Columns[colId].Type)
+		if colDataType >= memCom.ArrayBool && colDataType <= memCom.ArrayInt64 {
```

Comment:
maybe reuse `IsArrayType()`?

Reply:
nice catch, forget this function myself :)